lib/menu.rb

```diff
@@ -0,0 +1,39 @@
+class Menu
+  attr_reader :dishes
+  
+  ERROR_DISH_EXISTS = "Error: that dish number already exists"
+  ERROR_DISH_NOT_EXISTS = "Error: that dish is not on the menu"
+  
+  def initialize 
```

Comment:
Try and use Dependency Injection.
`def initialize(dishes = Array.new)
    @dishes = dishes
end
`